charts/fybrik/templates/fybrik-blueprints-role.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
```

Comment:
Why was the corresponding rolebinding deleted?

Reply:
Mistake, fix it, thanks